src/Markup/Avalonia.Markup.Xaml/Context/AvaloniaTypeFeatureProvider.cs

```diff
@@ -19,13 +10,22 @@
 using Avalonia.Media.Imaging;
 using Avalonia.Metadata;
 using Avalonia.Styling;
+using OmniXaml;
+using OmniXaml.Builder;
+using OmniXaml.TypeConversion;
+using OmniXaml.TypeConversion.BuiltInConverters;
+using OmniXaml.Typing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
 using OmniMetadata = OmniXaml.Typing.Metadata;
```

Comment:
Could you resort the usings? You need to enable the "Tools -> Options -> Text Editor -> C# -> Advanced -> Please System directives first when sorting usings" option in VS.